src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+/** SmallCityService object representing all components of the webapp **/
+public class SmallCityService {
+  
+  private User user;
+  private List<Listing> businessesList;
```

Comment:
I also mean removing the indication that it's a list from the variable. :)

Reply:
Oh yep that would make a lot more sense! Fixed!